dhis-2/pom.xml

```diff
@@ -623,7 +623,7 @@
       <dependency>
         <groupId>org.hisp.dhis.rules</groupId>
         <artifactId>rule-engine</artifactId>
-        <version>2.0.34.0-SNAPSHOT</version>
+        <version>2.0.1-SNAPSHOT</version>
```

Comment:
shouldn't it be `2.0.35.0.`

Reply:
We should really talk about this and have a common strategy.
In my opinion, Rule-engine versioning should not follow DHIS2-core versioning, it should be completely independent.

Reply:
Agree, it must be independent.